fuzz/client.c

```diff
@@ -25,6 +26,23 @@ extern int rand_predictable;
 /* unused, to avoid warning. */
 static int idx;
 
+#define FUZZTIME 1485898104
+
+#ifdef _WIN32
+# ifdef _TIME64_T_DEFINED
+time64_t _time64(time64_t *t)
+# else
+time_t _time(time_t *t)
```

Comment:
> if you enumerate symbols in 32-bit dll, you will note _time

Not quite. If you *compile*  C code with call to `time` and enumerate symbols in .obj, you'll note reference `_time`... Yeah, it's mess...